cfn-guard-rulegen-lambda/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cfn-guard-rulegen-lambda"
+version = "0.1.0"
```

Comment:
Can you set this to 0.5.2?  We're keeping them all the same across all artifacts.